nvm.sh

```diff
@@ -2682,6 +2682,41 @@ nvm() {
         return 127
       fi
 
+      # Install version if not available
+      if ! nvm ls "$PROVIDED_VERSION" &>/dev/null; then
+        tries_global=3
+        installed_flag="/tmp/.nvm_success"
+
+        while [ $tries_global -gt 0 ]; do
+          ( nvm install "$PROVIDED_VERSION" && touch $installed_flag ) &
+          proc_id="$!"
+          #wait $proc_id
+          #result=$?
+
+          # Check every 2 seconds if the installation finished,
+          # if it did, install the needed npm packages and exit
+          tries_install=10
+          while [ $tries_install -gt 0 ]; do
+            if [ -f $installed_flag ]; then
+              npm install -g grunt-cli bower
```

Comment:
Excuse me, I clicked too soon when commiting the changes to my fork. I need better glasses. :nerd_face: